File: src/test/java/application/browser/VisibleLibraryHelper.java
package application.browser;

import org.openqa.selenium.By;
import org.openqa.selenium.WebElement;

import com.matag.player.PlayerType;

public class VisibleLibraryHelper extends AbstractCardContainerHelper {
  private final PlayerType playerType;

  public VisibleLibraryHelper(MatagBrowser libraryHelper, PlayerType playerType) {
    super(libraryHelper);
    this.playerType = playerType;
  }

  @Override
  protected WebElement containerElement() {
    return matagBrowser.findElement(By.id(playerType.name().toLowerCase() + "-library"))
      .findElement(By.className("visible-cards"));
  }
}

File: src/test/kotlin/application/browser/VisibleLibraryHelper.kt
package application.browser

import com.matag.player.PlayerType
import org.openqa.selenium.By
import org.openqa.selenium.WebElement
import java.util.*

class VisibleLibraryHelper(libraryHelper: MatagBrowser?, private val playerType: PlayerType) :
    AbstractCardContainerHelper(libraryHelper) {
    override fun containerElement(): WebElement {
        return matagBrowser.findElement(By.id(playerType.name.lowercase(Locale.getDefault()) + "-library"))
            .findElement(By.className("visible-cards"))
    }
}
